### ISSUE TITLE
Fast Leaf Decay Integration

### ISSUE THREAD

**Issue by thelapisfox** [ldtteam/minecolonies-features#415]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [x] I made sure that **this change request is not a duplicate of any existing change request.**

### Context 

- Minecraft Version: 1.16.5
- MineColonies Version:  0.14.134-alpha
- Related Mods and their Versions:
Fast Leaf Decay: v25

## Is your feature request related to a problem?

When Fast leaf Decay is installed, the lumberjack painstakingly destroys each and every leaf block. This takes up a lot of time. Also, once the leaves do break away, any logs left behind get ignored.

## Enhancement or Change description

The change I am suggesting is adding in a switch where, if the mod detects that one has fast leaf decay installed, the lumberjack ai switches to only breaking log blocks. This would also fix the other issue as well and ensure they get all log blocks in their area.

### Notes or related things

---
## Viewers

* Add a thumbs-up to the change request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by ravenbuilder934** [comment]
#135, and also, sometimes foresters *need* to break leaves to get to the logs (which can happen if the trees are low enough to the ground).

**Comment by thelapisfox** [comment]
Mine seems to be a little different. I've noticed it starts by breaking the leaves in it's way and then begins breaking from the top down going each level at a time. Only problem with that is that it seems if it ends up clearing all the leaves from around some logs, it refuses to clean up those logs, thereby refusing to replant the sapling as the space for it currently has a log in it. This forces me to have to go over, remove those logs myself, and replant a sapling myself as well. Now, this doesn't happen very often, but with my spruce trees, it seems to happen enough to be annoying. I figured by having it the leaves no within 2 blocks if the ground, it would, at the very least, get rid of this issue.
